package.json

```diff
@@ -23,15 +23,15 @@
   },
   "homepage": "https://github.com/silverstripe/login-forms#readme",
   "dependencies": {
-    "bootstrap": "^5.2.0",
-    "@popperjs/core": "^2.11.6"
+    "@popperjs/core": "^2.11.6",
+    "bootstrap": "^5.2.0"
```

Comment:
Change in sort order must have been done automatically by yarn. All I did was `yarn upgrade @silverstripe/webpack-config@^3 && yarn build`.